### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by shijiesheng**
**What changed?**

See changelog

**Why?**

**How did you test it?**

internally tested with canary test D11792027

**Potential risks**

### CODE REVIEW

**Review by neil-xie — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.1.0] - 2023-11-06
+### Added
+- Added new poller thread pool usage metrics #1275
```
Comment (taylanisikdemir) [inline CHANGELOG.md:11]: `concurrent-task-running` metric is not reporting correct numbers. I told @timl3136 about it. let's fix the metrics before we release 1.1.0.

Comment (shijiesheng) [reply CHANGELOG.md]: included the rename #1291. Thank you for catching that

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added metrics tag workflowruntimelength in workflow context #1277
+- Added WithTaskListMapper and WithWorkflowTasklistMapper utils #1286
+
+### Changed
+- Updated idl version
+- Improved retrieval of binaryChecksum #1279
+
+### Fixed
+- Fixed error log #1284
+- Fixed workflow interceptor is not propagated correctly for child workflows #1289
```
Comment (Groxx) [inline CHANGELOG.md:21]: out of date with the PR title, as this is a test-only thing

Comment (shijiesheng) [reply CHANGELOG.md:21]: ah good catch

Comment (taylanisikdemir) [reply CHANGELOG.md:22]: Still out of date with PR title. missing test-only aspect

**Review by taylanisikdemir — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.1.0] - 2023-11-06
+### Added
+- Added new poller thread pool usage metrics #1275 #1291
+- Added metrics tag workflowruntimelength in workflow context #1277
+- Added DisableStrictNonDeterminismCheck worker option #1288
```
Comment (taylanisikdemir) [inline CHANGELOG.md:13]: this is a no-op change until I merge #1281. can be removed from changelog

**Pull request merged**